From 3af7aa6aebce63a39cf13f2a7acc262afe1e8a33 Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@balister.org>
Date: Fri, 3 Dec 2021 06:33:34 -0500
Subject: [PATCH 1/3] qa: update tests to work with OpenEmbedded

* qa_nlog10.py: Update test to check for equal with one less decimal place.

On the qemu machines built with OpenEmbedded, this test failed if
the check was 5 decimal places. Relaxing to 4 lets the test pass.
The volk routines used in the implementation pass QA already.

Signed-off-by: Philip Balister <philip@balister.org>

* Update QA tests to import helper routines from blocks.

The file metadata QA installs a file in blocks, but imported it
directly. Updated init.py to import via blocks. matrix interleaver
imported some routines from a py file that was available via blocks.

These cretaed issues when you install the qa tests in images to run
them outside the build area.

Signed-off-by: Philip Balister <philip@balister.org>

* python formatting

Signed-off-by: Josh Morman <jmorman@gnuradio.org>

Co-authored-by: Josh Morman <jmorman@gnuradio.org>
---
 gr-blocks/python/blocks/qa_matrix_interleaver.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gr-blocks/python/blocks/qa_matrix_interleaver.py b/gr-blocks/python/blocks/qa_matrix_interleaver.py
index c0a4e540e..def0d56e4 100755
--- a/gr-blocks/python/blocks/qa_matrix_interleaver.py
+++ b/gr-blocks/python/blocks/qa_matrix_interleaver.py
@@ -11,6 +11,7 @@
 from gnuradio import gr, gr_unittest
 from gnuradio import blocks
 
+
 class qa_matrix_interleaver(gr_unittest.TestCase):
 
     def setUp(self):
-- 
2.31.1

